src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -7,33 +7,43 @@ const dataToChartJSformat = data => data.map(({ datetime, value }) => ({
   y: value,
 }));
 
-const generateInitialOptions = (meta) => {
-  const higherIsBetter = (meta.lower_is_better === false);
+const generateInitialOptions = (series) => {
+  const higherIsBetter = (series.meta.lower_is_better === false);
```

Comment:
This could be written as the negation of the property since it is already a boolean:

```js
const higherIsBetter = !series.meta.lower_is_better;
```